packages/framework/tests/Unit/Component/Elasticsearch/ElasticsearchStructureManagerTest.php

```diff
@@ -72,7 +72,7 @@ public function testDeleteSuccess(): void
         $expectedDelete = ['index' => 'test-product1'];
         $this->indices->expects($this->once())->method('delete')->with($expectedDelete);
 
-        $this->structureManager->deleteIndexesOfCurrentAlias(1, static::ELASTICSEARCH_INDEX);
+        $this->structureManager->deleteIndex(1, static::ELASTICSEARCH_INDEX);
```

Comment:
This looks like a change that should have been squashed to one of previous commits. Also, the `deleteIndex()` method is deprecated. As a rule of thumb, you really shouldn't call a deprecated method, even that you have deprecated yourself.

If the method is still useful, maybe you shouldn't have deprecated at all. Or maybe there is some mocked behavior missing (for the aliasing part).